src/utils/props.js

```diff
@@ -63,13 +63,15 @@ export const getTextureFromProps = (
   if (image) {
     invariant(typeof image === 'string', elementType + ' image needs to be a string, got `%s`', typeof image)
     const args = [image]
+    // TODO: fix this, it seems crossorigin is not the second args, but in options.resourcesOptions.crossorigin
```

Comment:
@inlet

Reply:
btw if we need to change that, maybe doing this in an other PR once this one is merge would be better

Reply:
Awesome, create another PR and I'll merge it 👍

Reply:
https://github.com/inlet/react-pixi/issues/121

Reply:
Btw, I see that cross-origin defaults to true now, so we can get rid of this prop entirely. If you'd like to provide extra resource options you can always use the texture prop like so:

```jsx
<Sprite texture={
  PIXI.Texture.from('./image.png', { 
    resourceOptions: { scale: 2 } 
  })} 
/>
```